runtime/onert/core/src/util/shapeinf/Squeeze.cc

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "util/ShapeInference.h"
+
+namespace onert
+{
+namespace shape_inference
+{
+
+ir::Shape inferSqueezeShape(const ir::Shape &in_shape, const ir::operation::Squeeze::Param &param)
+{
+    const int ndims = param.ndim;
+    const int* squeeze_dims = param.dims;
+    bool should_squeeze[8] = {false};
+    int num_squeezed_dims = 0;
+    if (ndims == 0) {//전체 dims이 1인것 제거
+        for (int idx = 0; idx < in_shape.rank(); ++idx)
+        {
+            if (in_shape.dim(idx) == 1) {
+                should_squeeze[idx] = true;
+                ++num_squeezed_dims;
+            }
+        }
+    } else {
+        for (int idx = 0; idx < ndims; ++idx) {
+            int current = squeeze_dims[idx];
+            if (current < 0) {
+                current = squeeze_dims[idx] + in_shape.rank();
+            } else {
+                current = squeeze_dims[idx];
+            }
+
+            if (!(current >= 0 && current < in_shape.rank() && in_shape.dim(current) == 1)) {
```

Comment:
It use in_shape.rank() frequently, So how about using it as a variable?
`
int rank = in_shape.rank();
`

Reply:
okay
Modify